tests/test_agent/test_http_api_state.py

```diff
@@ -38,6 +38,10 @@ class TestStop(BaseAPITestCase):
     URI = "/stop"
     CLASS = Stop
 
+    def prepare_config(self):
+        super(TestStop, self).prepare_config()
+        config.update(run_control_file="/tmp/pyfarm/agent/should_be_running")
```

Comment:
Use `self.create_file()` in the tests.  It's cross-platform and will manage the file handle.